src/vscode-bicep/src/commands/generateParams.ts

```diff
@@ -34,6 +35,15 @@ export class GenerateParamsCommand implements Command {
         }
       );
       this.outputChannelManager.appendToOutputChannel(generateParamsOutput);
+
+      const filePath = path.parse(documentUri.fsPath);
+
+      const openPath = vscode.Uri.parse(
+        `file://${filePath.dir}/${filePath.name}.parameters.json`
```

Comment:
better: 
var doc = await vscode.workspace.openTextDocument(openPath);
await vscode.window.showTextDocument(doc);

Reply:
that totally makes sense, thanks for the suggestion. created commit to update the code